spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -16,8 +16,9 @@
 
 package org.springframework.cloud.gcp.storage;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import java.util.logging.Logger;
```

Comment:
FYI the reason I moved away from slf4j is checkstyle:

[ERROR] src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java:[21,1] (imports) IllegalImport: Import from illegal package - org.slf4j.Logger.
[ERROR] src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java:[22,1] (imports) IllegalImport: Import from illegal package - org.slf4j.LoggerFactory.

I'll move into Spring commons logging.